dom.js

```diff
@@ -0,0 +1,180 @@
+let a = null;
+let b = null;
+let action = null;
+let lastAction = null;
+
+
+
+const calculateButton = document.getElementById('calculate');
+const addButton = document.getElementById('add');
+const multiplayButton = document.getElementById('multiplay');
+const subButton = document.getElementById('sub');
+const divisionButton = document.getElementById('division');
+const ceButton = document.getElementById('ce');
+const cButton = document.getElementById('c');
+
+const sqrButton = document.getElementById('sqr');
+const sqrtButton = document.getElementById('sqrt');
+const invertButton = document.getElementById('invert');
+const piNumberButton = document.getElementById('pi');
+const eNumberButton = document.getElementById('e');
+const resultScr = document.getElementById('result');
+
+const num1Button = document.getElementById('num1');
+const num2Button = document.getElementById('num2');
+const num3Button = document.getElementById('num3');
+const num4Button = document.getElementById('num4');
+const num5Button = document.getElementById('num5');
+const num6Button = document.getElementById('num6');
+const num7Button = document.getElementById('num7');
+const num8Button = document.getElementById('num8');
+const num9Button = document.getElementById('num9');
+const num0Button = document.getElementById('num0');
+const dotButton = document.getElementById('dot');
+
+function digitKey(digit) {
+    if (resultScr.value === '0') {
+        resultScr.value = digit;
+    } else {
+        resultScr.value = resultScr.value + digit;
+    }
+} 
+
+
+
+num1Button.addEventListener('click', () =>{

```

Comment:
Could you read this article: https://dmitripavlutin.com/javascript-event-delegation/